UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -222,6 +222,11 @@ public void SubSetBodyPart(BodyPart Body_Part, string path)
 
 	public IEnumerator WaitForPlayerinitialisation()
 	{
+		yield return null;
+		yield return null;
+		yield return null;
```

Comment:
No better way?

Reply:
oh yeah,  I remember why now, it's because the populate is on the body parts would get populated before server start from a awake, and then they will get populated again from Server start

Reply:
double remember now, xD
it's actually because you'd have to wait a few frames for body parts to add themselves to the storage before you could add clothing to the

Reply:
Hmm ok. Well, a comment would be very valuable here. Very difficult to find the purpose just by code alone.

Reply:
ok